File: Java/Solutions/2.1 Simple Calculations/_10_RadiansToDegrees.java
import java.util.Scanner;

public class _10_RadiansToDegrees {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double radians = Double.parseDouble(scanner.nextLine());
        long degree = Math.round(radians * 180 / Math.PI);
        System.out.println(degree);
    }
}

File: JavaScript/Solutions/02.Simple-Calculations/10-radians-to-degrees.js
function radiansToDegrees([arg1]) {
    let radians = Number(arg1);
    let degrees = radians * 180 / Math.PI;
    console.log(Math.round(degrees));
}

radiansToDegrees([3.1416]);
radiansToDegrees([6.2832]);
radiansToDegrees([0.7854]);
radiansToDegrees([0.5236]);
